### DISCUSSION TITLE
Documentation website

### DISCUSSION THREAD

**Discussion by macklinu** [sergiodxa/remix-utils#280] [Ideas]
Hello, since this library has so many useful functions, navigating through a README to explore all the APIs and examples leaves a little bit to be desired. I would be happy to contribute a PR for a documentation website for remix-utils if that's something you'd be interested in (for example, a [Starlight Astro docs site](https://starlight.astro.build/)).

<img width="1080" alt="image" src="https://github.com/sergiodxa/remix-utils/assets/2344137/0735aabe-0646-4a8b-b9d7-c2d4c1585176">

Or perhaps a simpler first step would be a PR for adding an autogenerated table of contents to the top of the README for jumping around to different API doc sections using a package like [doctoc](https://github.com/thlorenz/doctoc).

Any preferences? Thanks!

**Comment by sergiodxa** [comment]
GitHub already auto generates a table of content https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

For the website, I have already considered, but I never have the time to build it, and if I do it I would deploy a Remix app to Cloudflare Workers that fetches the README.

**Reply by macklinu** [reply]
Ah good point about the generated TOC! I didn't see that on GitHub iOS but do on the web, so that's great. If you do end up wanting to explore an actual website, feel free to mention me - would love to help out. 😄
